src/main/jbake/content/docs/user/quick-start.adoc

```diff
@@ -156,6 +156,13 @@ TIP: Optional: A single Hawkular WildFly Agent instance can also manage multiple
 Now you can start the managed WildFly instance, keeping in mind that if you are following this QuickStart your Hawkular Server is probably already running on port 8080, so you need to run this instance on a different port by starting the server with `sh bin/standalone.sh -Djboss.socket.binding.port-offset=100` for instance.
 Now if you have correcty set up the agent, it will report into your Hawkular Server at http://localhost:8080/hawkular-ui/app/app-list.
 
+To push data about Application Performance Management / Business Transaction Management you need to install the (second) relative agent available in `btm` folder. Open the `btm\setenv.sh` file and set the correct URL user/pass and do:
+----
+source btm\setenv.sh
```

Comment:
would you mind changing the backslash to normal slash? We are not windows haters, it's just, it's not supported on windows, afaik :]